### DISCUSSION TITLE
Auto mapping prisma model names to snake_case

### DISCUSSION THREAD

**Discussion by marina-ki** [blitz-js/blitz#2011] [Ideas]
I have mapped table names and column names to snake cases. Table names are pluralized.

```
model User {
  id             Int       @id @default(autoincrement())
  createdAt      DateTime  @default(now()) @map("created_at")
  updatedAt      DateTime  @updatedAt @map("updated_at")
  name           String
  email          String    @unique
  hashedPassword String  @map("hashed_password")
  role           Role  @default(user)
  @@map("users")
}
```

I think this is a common way to name tables, so it would be nice to have an option to map them automatically.

ex:
`$ blitz g model user --map`

**Comment by flybayer** [comment]
That's definitely the traditional way with Rails :) I've been using camelCase with prisma and am happy so far.

I'm curious if others want this too.

**Reply by marina-ki** [reply]
I think Postgres should use lower case. Postgres folds identifiers to lowercase, unless you double-quote your identifiers. To make Postgres operations easier, use lower_snake

**Comment by flybayer** [comment]
I think it may be better to use a config item in `blitz.config.js` instead of a `--map` flag
